### PR TITLE
Request to add PyTNG package.

### PR DESCRIPTION
**Pull request by hmacdope**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hmacdope** [comment]
@richardjgowers  @orbeckst, can you confirm that you are happy to be listed as maintainers? :)

**Comment by orbeckst** [comment]
Sure!

**Comment by hmacdope** [comment]
@conda-forge/staged-recipes ready to roll I think.

**Comment by richardjgowers** [comment]
Yup

**Comment by hmacdope** [comment]
@orbeckst are you able to ping @conda-forge/help-python, as a first time contributor I don't think I can?

**Comment by orbeckst** [comment]
@conda-forge/staged-recipes please review and merge.

File: recipes/pytng/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+{% set version = "0.2.2a1" %}
+{% set sha256 = "688fb4afcc5630bd98553834b5dbd69eb04b83e2fc23f4322986f52add5bcfda" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/pytng/meta.yaml:11]: You don't need the `fn` line

**Comment by orbeckst** [comment]
@hmacdope , the [README](https://github.com/conda-forge/staged-recipes/blob/master/README.md) said to ping `@conda-forge/staged-recipes` so I tried this first. Is there a reason for pinging  `@conda-forge/help-python`? — I'll happily do it, I just don't want to add noise to people's inboxes.

File: recipes/pytng/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
+    - zlib
+
+  run:
+    - python
+    - cython
```
Comment (synapticarbors) [inline recipes/pytng/meta.yaml:34]: Do you need `cython` at run time?

**Comment by hmacdope** [comment]
@orbeckst no I just tried help-python as I was unable to ping conda-forge/staged-recipes. Thanks for sorting out.

File: recipes/pytng/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
```
Comment (synapticarbors) [inline recipes/pytng/meta.yaml:29]: If you're building against `numpy` see:

https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

**Review by orbeckst — APPROVED** [review]
LGTM

**Comment by hmacdope** [comment]
@synapticarbors thanks for the feedback. I think I have fixed it all up.

File: recipes/pytng/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+{% set name = "pytng" %}
+{% set version = "0.2.2a1" %}
```
Comment (ocefpaf) [inline recipes/pytng/meta.yaml:2]: This is an alpha release and should probably follow https://github.com/conda-forge/cfep/blob/master/cfep-05.md

However, we cannot do that in staged-recipes. Also. this package does not have a stable release yet. So let's merge it and ask the maintainers to implement CFEP-05 in the feedstocks.

**Pull request merged**